.travis.yml

```diff
@@ -0,0 +1,22 @@
+################################################
+#  LangSandbox by Federico Tomassetti:         #
+#  https://github.com/ftomassetti/LangSandbox  #
+################################################
+
+dist: trusty
+
+git:
+  depth: false
+
+install:
+  - npm install -g eclint
+
+script:
+  # ==============================================
```

Comment:
We should probably also run tests using `./gradlew check`

Reply:
I would restrict this PR to EditorConfig features, and in a separate commit we can add whichever other Travis jobs are needed (also, I suggest running them separately, so we get job-specific build-errors). 

But I don't work much with Gradle (except as a consumer of Gradle projects), so I don't know much about how it works.

Reply:
Ok, it makes sense